### PR TITLE
[CD][Dashboards]enable the anomalyDetection osd plugin into build workflo

### PR DESCRIPTION
**Pull request by Tengda-He**
### Description
Enable the  anomalyDetection osd plugin into build workflow.
Test is in progress, pending development environment setup. Seeking early feedback.
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/652
 
### Check List
- [X ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dblock** [comment]
As long as YAML can be corrected and you can run the build beginning to end without errors this LGTM.

**Review by seraphjiang — COMMENTED** [review]
DCO check failed.

https://wiki.linuxfoundation.org/dco

File: bundle-workflow/scripts/default/opensearch-dashboards/build.sh [hunk 1]
```diff
@@ -68,3 +68,4 @@ echo "BUILD RELEASE ZIP FOR $PLUGIN_NAME"
 (cd ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME && yarn plugin-helpers build)
 echo "COPY $PLUGIN_NAME.zip"
 cp -r ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME/build/$PLUGIN_NAME-$VERSION.zip $OUTPUT/plugins/
+rm -rf ../OpenSearch-Dashboards/plugins/$PLUGIN_NAME
```
Comment (dblock) [inline bundle-workflow/scripts/default/opensearch-dashboards/build.sh:71]: Why is this needed?

Comment (Tengda-He) [reply bundle-workflow/scripts/default/opensearch-dashboards/build.sh:71]: without this I observe error below during plugin build. Synced with rocky, seems we need to delete the plugin folder content after the artifact is generated, otherwise it will impact the following plugin build.
```
    The conflicting dependencies are:

        cypress
          ^6.0.0 => opensearch-alerting-dashboards
          ^5.0.0 => anomaly-detection-dashboards
        formik
          ^2.2.6 => opensearch-alerting-dashboards
          ^2.2.5 => anomaly-detection-dashboards
```

Comment (dblock) [reply bundle-workflow/scripts/default/opensearch-dashboards/build.sh:71]: I think we need an issue to make these builds more independent. I am not sure how this would work, but reading this it seems like an _installation_ of a plugin pollutes another plugin installation?

Comment (Tengda-He) [reply bundle-workflow/scripts/default/opensearch-dashboards/build.sh:71]: yes one pollutes another. I will create an issue to track this.
